mzPlot/mzPlot.csproj

```diff
@@ -1,7 +1,7 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
```

Comment:
Okay, makes sense! Maybe this is one of the things they worked on fixing with net5.0. It's now listed in a different grouping because of the net5.0/net5.0-windows split, rather than listing them both in the netcoreapp3.x framework. https://github.com/smith-chem-wisc/mzLib/pull/569/files#diff-e99b0c47506e01b3af18d4d0a4605bcc3f853bee6236c867e605255267ea9c39R24